### PR TITLE
Project 1: basic page structure.

### PR DESCRIPTION
**Pull request by jecode93**
In this milestone I:

- Create an npm project with Webpack.
- Write plain HTML markup with minimum styling (just to make the layout)

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/* stylelint-disable indentation */
+/* stylelint-disable csstree/validator */
+
```
Comment (Strangeal) [inline src/css/styles.css]: - [x] Great job on completing the first step of your leaderboard project as specified in the requirements! 👏 However, I would like to emphasize the importance of keeping `stylelint` enabled. Stylelint plays a crucial role in helping you write industry-standard styles for your project, ensuring consistency and maintainability.

I understand that you mentioned disabling `stylelint` in the attached PR, but I would recommend against doing so. Instead, let's work together to address these errors and improve your code. I'm here to help you learn and understand how to fix them, which will ultimately enhance your coding skills.

To address these errors, you can use the `--fix` flag, which automatically corrects simple style errors. For stylelint-specific issues, you can utilize the following command:

```bash
npx stylelint "**/*.{css,scss}" --fix
```

By running this command, `stylelint` will fix the indentation issues you have automatically, making your styles error-free. 😊😊

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @jecode93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/styles.css (hunk 1 above)
Comment (jecode93) [reply src/css/styles.css]: Hello @Strangeal 
Thank you so much for your review!

I didn't know how to fix that without disabling the linter rule, thank you very much.

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @jecode93👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented the changes requested by the previous code reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**